### ISSUE TITLE
GTA-V not launching

### ISSUE THREAD

**Issue by aniketfuryrocks** [flavioislima/HeroicGamesLauncher#68]
The game starts with `installing launcher` and then gives a popup 'Unable to launch the game, please verify game files.'

```log
[cli] INFO: Logging in...
[Core] INFO: Trying to re-use existing login session...
[cli] INFO: Checking for updates...
[Core] INFO: Getting authentication token...
[cli] INFO: Launching 9d2d0eb64d5c44529cece33fe2a46482...
ERROR: ld.so: object 'libgamemodeauto.so.0' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
ERROR: ld.so: object 'libgamemodeauto.so.0' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
ERROR: ld.so: object 'libgamemodeauto.so.0' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
ERROR: ld.so: object 'libgamemodeauto.so.0' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
wine: Unhandled page fault on read access to 00208228 at address 7BC2966F (thread 010c), starting debugger...
010c:err:seh:NtRaiseException Unhandled exception code c0000005 flags 0 addr 0x7bc2966f
gamemodeauto: 
```

**Comment by flavioislima** [comment]
This message that heroic gives is quite generic for now. For the log it looks like a wine error, similar to #70. so you need to get help on some linux community because this is not related with heroic, but some missing dependency on your wine prefix. 
GTA V requires some things to run because of the rockstar launcher, so search on how to do it and it will probably work.

**Comment by aniketfuryrocks** [comment]
I installed everything, and it runs just fine, but it fails to launch login/authenticate when running through Lutris or Heroic Launcher. Works fine when I use Epic games launcher.

**Comment by wiryfuture** [comment]
> I installed everything, and it runs just fine, but it fails to launch login/authenticate when running through Lutris or Heroic Launcher. Works fine when I use Epic games launcher.

I tried myself and the Epic Games version is rather buggy, first I had issues where it wouldn't launch, it managed to launch with the lutris-5.7 version of wine but had keyboard input problems, and once I got into the game using directx10.1 or something, there was lighting bugs and very poor performance. I can confidently say that the epic games version of gtav doesn't work well with wine right now and you should stay away from it for now unless you have the time to do extensive testing with different configs.

Also, this issue can be closed because this isn't a HGL issue for sure.

**Comment by flavioislima** [comment]
Yeah, in this case try to show this game log on the linux community on reddit, maybe something is missing. I dont really have time now to test GTAV, maybe in the weekend I can and if so I'll post here if I made it work on Heroic.

But for now we can close the issue since this is more like a wine support.

**Comment by flavioislima** [comment]
@aniketfuryrocks I was able to run GTA V with Heroic. But I had to run it without gamemode, for some reason is having issues.
After the first run, it installed the Rockstar launcher and then said that the game data was corrupted. I closed, opened the game again and the rockstar launcher updated and asked me to login. After login it didnt find the game. So I closed it again, opened the game again and it worked. It went straight into the game.

So try it, it might work. I used Wine Lutris 6.0 rc1 with a clean prefix. Haven't installed or configured anything with winecfg or winetricks.

**Comment by jarrard** [comment]
It launches for me after a few tries but I jsut get a video of a broken glass sound and it closes again. Not sure why.

**Comment by jarrard** [comment]
game doesn't appear to save video/graphic settings correctly. Damn

**Comment by flavioislima** [comment]
> It launches for me after a few tries but I jsut get a video of a broken glass sound and it closes again. Not sure why.
> 
> Its possible proton to blame? but it does launch, it just doesn't say launched.
> 
> Nope, just get the broken glass intro all the time no matter what I try.
> Probably corrupted files, not downloading 100GB again. We need a way to verify files, otherwise this is just pointless.
> 
> Can do repair in cmd line via the below. Not sure if it will fix my game but it did find some broken files.
> 
> legendary verify-game 9d2d0eb64d5c44529cece33fe2a46482
> legendary repair 9d2d0eb64d5c44529cece33fe2a46482
> 
> This should be easy to implement. I believe I got corrupt files because the download took 10 tries to get going (lovely AU internet)

Repair/verify is in the works. For now, you need to use Legendary.

There is a lot of things that are 'easy' to implement, but there is a lot of things to implement, and for now, only me doing this. 
Heroic is not even 1 month old, with time we will have all these features. 😃

**Comment by flavioislima** [comment]
> game doesn't appear to save video/graphic settings correctly. Damn
> 
> Update: Apparently you've got to run the benchmark and not let it exit the game first or settings reset. Some gamebug. Also apparently saving settings when in offline story mode works. Just not at main menu without even starting a game or benchmark.
> 
> Sadly game has black or white texture issues for me. So its broken. But at least I got it working with this launcher app.

GTAV have several issues on Linux, I think wine works better than proton but that will depend on your machine as well, drivers, etc.

**Comment by flavioislima** [comment]
Closing this one since repair games was implemented on 1.2.0.
